packages/components/organisms/f-checkout/src/components/_tests/Error.test.js

```diff
@@ -42,4 +68,71 @@ describe('Error', () => {
         // Assert
         expect(description.text()).toMatchSnapshot();
     });
+
+    describe('methods', () => {
+        describe('`redirectToMenu` ::', () => {
+            const oldWindowLocation = window.location;
+
+            beforeAll(() => {
+                delete window.location;
+
+                window.location = Object.defineProperties(
+                    {},
+                    {
+                        ...Object.getOwnPropertyDescriptors(oldWindowLocation),
+                        assign: {
+                            configurable: true,
+                            value: jest.fn()
+                        }
+                    }
+                );
+            });
+
+            afterAll(() => {
+                window.location = oldWindowLocation;
+            });
+
+            describe('when invoked', () => {
+                it('should redirect the customer back to the menu page so the user can navigate back from the error', () => {
+                    // Act
+                    wrapper.vm.redirectToMenu();
+
+                    // Assert
+                    expect(window.location.assign).toHaveBeenCalledWith('menu-jason-1');
+                });
+
+                describe('AND errorFormType is accessForbiddenError', () => {
+                    beforeEach(async () => {
+                        await wrapper.setProps({ errorFormType: CHEKOUT_ERROR_FORM_TYPE.accessForbidden });
+                    });
+                    describe('AND a menu basket cookie does not exist', () => {
+                        it('should not delete the cookie', () => {
+                            // Arrange
+                            const cookieRemoveSpy = jest.spyOn(wrapper.vm.$cookies, 'remove');
+
+                            // Act
+                            wrapper.vm.redirectToMenu();
+
+                            // Assert
+                            expect(cookieRemoveSpy).not.toHaveBeenCalled();
+                        });
+                    });
+
+                    describe('AND a menu basket cookie already exists', () => {
+                        it('should delete the cookie', () => {
+                            // Arrange
+                            jest.spyOn(wrapper.vm.$cookies, 'get').mockReturnValue(() => 'je-mw-basket');
+                            const cookieRemoveSpy = jest.spyOn(wrapper.vm.$cookies, 'remove');
```

Comment:
can this go in the beforeEach for this and the test above?

Reply:
Yes that would work. @dandel10n would you be able to do this in this pr or another? If not I can fix in another PR myself.

Reply:
@FayeCarter  @kevinrodrigues  Can we apply it in another PR, as adding the change here would trigger all the builds again and it takes ages?